osconfig_tests/test_suites/package_management/package_management.go

```diff
@@ -112,44 +108,197 @@ func TestSuite(ctx context.Context, tswg *sync.WaitGroup, testSuites chan *junit
 	logger.Printf("Finished TestSuite %q", testSuite.Name)
 }
 
-func runCreateOsConfigTest(ctx context.Context, testCase *junitxml.TestCase, logger *log.Logger) {
+func runCreateOsConfigTest(ctx context.Context, testCase *junitxml.TestCase, testSetup *packageManagementTestSetup, logger *log.Logger) {
 
-	osConfig := &osconfigpb.OsConfig{
-		Name: "create-osconfig-test",
+	oc := &osconfigserver.OsConfig{
+		&osconfigpb.OsConfig{
+			Name: "createosconfig-test-osconfig",
+		},
 	}
 
-	defer osconfigserver.CleanupOsConfig(ctx, testCase, logger, osConfig)
+	logger.Printf("create osconfig request:\n%s\n\n", dump.Sprint(oc))
 
-	req := &osconfigpb.CreateOsConfigRequest{
-		Parent:   "projects/compute-image-test-pool-001",
-		OsConfig: osConfig,
-	}
+	parent := fmt.Sprintf("projects/%s", testProject)
+	res, err := osconfigserver.CreateOsConfig(ctx, logger, oc, parent)
 
-	logger.Printf("create osconfig request:\n%s\n\n", dump.Sprint(req))
+	defer cleanuposconfig(ctx, testCase, logger, oc)
 
-	res, err := osconfigserver.CreateOsConfig(ctx, logger, req)
 	if err != nil {
 		testCase.WriteFailure("error while creating osconfig:\n%s\n\n", err)
 	}
 
 	logger.Printf("CreateOsConfig response:\n%s\n\n", dump.Sprint(res))
 }
 
+func runPackageInstallTest(ctx context.Context, testCase *junitxml.TestCase, testSetup *packageManagementTestSetup, logger *log.Logger) {
+
+	osConfig, err := osconfigserver.JsonToOsConfig(packageInstallTestOsConfigString, logger)
+
+	if err != nil {
+		testCase.WriteFailure("error while converting json to osconfig: \n%s\n", err)
+		return
+	}
+
+	oc := &osconfigserver.OsConfig{OsConfig: osConfig}
+
+	parent := fmt.Sprintf("projects/%s", testProject)
+	osconfigserver.CreateOsConfig(ctx, logger, oc, parent)
+
+	if err != nil {
+		testCase.WriteFailure("error while creating osconfig: \n%s\n", err)
+		return
+	}
+
+	defer cleanuposconfig(ctx, testCase, logger, oc)
+
+	assignment, err := osconfigserver.JsonToAssignment(packageInstallTestAssignmentString, logger)
+	if err != nil {
+		testCase.WriteFailure("error while converting json to assignment: \n%s\n", err)
+		return
+	}
+
+	assign := &osconfigserver.Assignment{Assignment: assignment}
+
+	res, err := osconfigserver.CreateAssignment(ctx, logger, assign, parent)
+	if err != nil {
+		testCase.WriteFailure("error while creating assignment: \n%s\n", err)
+	}
+	_ = res
```

Comment:
if you don't care about `res`, then you can change line 162 to `_, err := ...` and remove this.

Reply:
done